module/applications/levelup/characterLevelup.mjs

```diff
@@ -280,11 +280,18 @@ export default class DhCharacterLevelUp extends LevelUpBase {
                                     break;
                                 case 'experience':
                                     if (!advancement[choiceKey]) advancement[choiceKey] = [];
+                                    const allExperiences = {
+                                        ...this.actor.system.experiences,
+                                        ...Object.values(this.levelup.levels).reduce((acc, level) => {
+                                            Object.keys(level.achievements.experiences).forEach(
```

Comment:
can we use a for loop here p.please? When I skimmed this, it looked like you were doing map on a quick skim and I had to double take.